### PR TITLE
update metadata of omega_slope

### PR DESCRIPTION
**Pull request by ninaburg**
## Purpose

Update metadata of `omega_slope` operator.

### Note

Fieldextra returns the parameter `etadot` for the `omega_slope` operator but we return w (Vertical Velocity (Pressure)). Because according to Fieldextra's description:
Convert from ECMWF etadot (i.e. deta/dt) to etadot*dp/deta

we infer that the returned parameter aligns with the COSMO definition of omega as Vertical Velocity (Pressure) (omega=dp/dt). In ECMWF, this vertical velocity is associated with the parameter w.

Additionally, Flexpart expects the GRIB code for "etadot" but converts the ID for "etadot" (ID: 77) to vertical velocity "w" (ID: 135), as shown in these code snippets: [link](https://github.com/MeteoSwiss/flexpart/blob/main/src/readwind_ecmwf.f90#L141C1-L144C10) and [link](https://github.com/MeteoSwiss/flexpart/blob/main/src/readwind_ecmwf.f90#L191-L194).

### CODE REVIEW

File: src/meteodatalab/operators/omega_slope.py [hunk 1]
```diff
@@ -30,7 +31,15 @@ def cumdiff(A, axis):
 def omega_slope(
     ps: xr.DataArray, etadot: xr.DataArray, ak: xr.DataArray, bk: xr.DataArray
 ):
-    """Compute omega slope."""
+    """Compute the omega slope.
+
+    Converts ECMWF etadot (deta/dt) to etadot * dp/deta, required by FLEXPART.
+
+    Note:
```
Comment (cfkanesan) [inline src/meteodatalab/operators/omega_slope.py:38]: Should be plural and without column
https://numpydoc.readthedocs.io/en/latest/format.html#notes

File: src/meteodatalab/operators/omega_slope.py [hunk 2]
```diff
@@ -30,7 +31,15 @@ def cumdiff(A, axis):
 def omega_slope(
     ps: xr.DataArray, etadot: xr.DataArray, ak: xr.DataArray, bk: xr.DataArray
 ):
-    """Compute omega slope."""
+    """Compute the omega slope.
```
Comment (cfkanesan) [inline src/meteodatalab/operators/omega_slope.py:34]: Could you also add the documentation for the parameters and return value?

File: src/meteodatalab/operators/omega_slope.py [hunk 3]
```diff
@@ -42,5 +51,9 @@ def omega_slope(
         / (dak_dz / pc.surface_pressure_ref + dbk_dz)
     ).reduce(cumdiff, dim="z")
 
-    res.attrs = etadot.attrs
-    return res
+    return xr.DataArray(
+        data=res,
+        attrs=metadata.override(
+            etadot.message, discipline=0, parameterCategory=2, parameterNumber=8
```
Comment (cfkanesan) [inline src/meteodatalab/operators/omega_slope.py:57]: Maybe there could be a comment indicating that this is the "Vertical velocity (pressure)"

File: tests/test_meteodatalab/test_flexpart.py [hunk 4]
```diff
@@ -89,6 +89,14 @@ def test_flexpart(data_dir, fieldextra):
         }
     )
 
+    assert ds_out["omega"].parameter == {
+        "centre": "ecmf",
```
Comment (cfkanesan) [inline tests/test_meteodatalab/test_flexpart.py:93]: This matches the output of eccodes?

**Review by cfkanesan — COMMENTED** [review]
Some minor comments but the overall changes look good to me. Thanks for researching this!

File: tests/test_meteodatalab/test_flexpart.py (hunk 4 above)
Comment (ninaburg) [reply tests/test_meteodatalab/test_flexpart.py:93]: Yes!! the "w" is missing in `ECMWF`, I don't know if it is on purpose. But this is retrieved with `metadata.as_namespace("parameter")` which is from `earthkit-data` so I don't think we can correct anything...

**Review by cfkanesan — APPROVED** [review]

**Pull request merged**